<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1320</width>
    <height>658</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="6,2,1">
      <item>
       <layout class="QVBoxLayout" name="viewfinderLayout_" stretch="2">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="viewfinderLayout">
          <item>
           <widget class="CusLabel" name="label_2">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="PaintLabel" name="label">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QTableWidget" name="tableWidget"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7" stretch="1">
        <property name="spacing">
         <number>1</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="widget_2" native="true">
              <widget class="QGroupBox" name="groupBox">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>0</y>
                 <width>121</width>
                 <height>131</height>
                </rect>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="title">
                <string/>
               </property>
               <widget class="QPushButton" name="camButton">
                <property name="geometry">
                 <rect>
                  <x>15</x>
                  <y>10</y>
                  <width>41</width>
                  <height>31</height>
                 </rect>
                </property>
                <property name="focusPolicy">
                 <enum>Qt::ClickFocus</enum>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton:checked{background-color: rgba(38, 166, 154,  255);  border: 1px solid rgb(124, 124, 124); border-radius:5px;}
QPushButton{background-color: rgba(84, 110, 122,  255);  border: 1px solid rgb(124, 124, 124); border-radius:5px;}
QPushButton:pressed{background-color: rgba(0, 77, 64,  255);  border: 1px solid rgb(124, 124, 124); border-radius:5px;}
QPushButton:disabled{background-color: rgba(0, 77, 64,  255);  border: 1px solid rgb(124, 124, 124); border-radius:5px;}</string>
                </property>
                <property name="text">
                 <string>CAM1</string>
                </property>
               </widget>
               <widget class="QPushButton" name="picButton">
                <property name="geometry">
                 <rect>
                  <x>15</x>
                  <y>50</y>
                  <width>91</width>
                  <height>31</height>
                 </rect>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton:checked{background-color: rgba(38, 166, 154,  255);  border: 1px solid rgb(124, 124, 124); border-radius:5px;}
QPushButton{background-color: rgba(84, 110, 122,  255);  border: 1px solid rgb(124, 124, 124); border-radius:5px;}
QPushButton:pressed{background-color: rgba(0, 77, 64,  255);  border: 1px solid rgb(124, 124, 124); border-radius:5px;}
QPushButton:disabled{background-color: rgba(0, 77, 64,  255);  border: 1px solid rgb(124, 124, 124); border-radius:5px;}</string>
                </property>
                <property name="text">
                 <string>图片</string>
                </property>
               </widget>
               <widget class="QPushButton" name="calibrationButton">
                <property name="geometry">
                 <rect>
                  <x>15</x>
                  <y>90</y>
                  <width>91</width>
                  <height>31</height>
                 </rect>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton:checked{background-color: rgba(38, 166, 154,  255);  border: 1px solid rgb(124, 124, 124); border-radius:5px;}
QPushButton{background-color: rgba(84, 110, 122,  255);  border: 1px solid rgb(124, 124, 124); border-radius:5px;}
QPushButton:pressed{background-color: rgba(0, 77, 64,  255);  border: 1px solid rgb(124, 124, 124); border-radius:5px;}
QPushButton:disabled{background-color: rgba(0, 77, 64,  255);  border: 1px solid rgb(124, 124, 124); border-radius:5px;}</string>
                </property>
                <property name="text">
                 <string>校准</string>
                </property>
               </widget>
               <widget class="QPushButton" name="camButton_2">
                <property name="geometry">
                 <rect>
                  <x>65</x>
                  <y>10</y>
                  <width>41</width>
                  <height>31</height>
                 </rect>
                </property>
                <property name="focusPolicy">
                 <enum>Qt::ClickFocus</enum>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton:checked{background-color: rgba(38, 166, 154,  255);  border: 1px solid rgb(124, 124, 124); border-radius:5px;}
QPushButton{background-color: rgba(84, 110, 122,  255);  border: 1px solid rgb(124, 124, 124); border-radius:5px;}
QPushButton:pressed{background-color: rgba(0, 77, 64,  255);  border: 1px solid rgb(124, 124, 124); border-radius:5px;}
QPushButton:disabled{background-color: rgba(0, 77, 64,  255);  border: 1px solid rgb(124, 124, 124); border-radius:5px;}</string>
                </property>
                <property name="text">
                 <string>CAM2</string>
                </property>
               </widget>
              </widget>
              <widget class="QGroupBox" name="groupBox_2">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>140</y>
                 <width>120</width>
                 <height>91</height>
                </rect>
               </property>
               <property name="title">
                <string/>
               </property>
               <widget class="QPushButton" name="loadFileButton">
                <property name="geometry">
                 <rect>
                  <x>15</x>
                  <y>10</y>
                  <width>91</width>
                  <height>31</height>
                 </rect>
                </property>
                <property name="font">
                 <font>
                  <family>微软雅黑</family>
                 </font>
                </property>
                <property name="text">
                 <string>载入流程文件</string>
                </property>
               </widget>
               <widget class="QPushButton" name="saveFileButton">
                <property name="geometry">
                 <rect>
                  <x>15</x>
                  <y>50</y>
                  <width>91</width>
                  <height>31</height>
                 </rect>
                </property>
                <property name="font">
                 <font>
                  <family>微软雅黑</family>
                 </font>
                </property>
                <property name="text">
                 <string>保存流程文件</string>
                </property>
               </widget>
              </widget>
              <widget class="QGroupBox" name="groupBox_3">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>240</y>
                 <width>120</width>
                 <height>291</height>
                </rect>
               </property>
               <property name="title">
                <string/>
               </property>
               <widget class="QPushButton" name="addButton">
                <property name="geometry">
                 <rect>
                  <x>15</x>
                  <y>10</y>
                  <width>91</width>
                  <height>31</height>
                 </rect>
                </property>
                <property name="font">
                 <font>
                  <family>微软雅黑</family>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton:checked{background-color: rgba(38, 166, 154,  255);  border: 1px solid rgb(124, 124, 124); border-radius:5px;}
QPushButton{background-color: rgba(84, 110, 122,  255);  border: 1px solid rgb(124, 124, 124); border-radius:5px;}
QPushButton:pressed{background-color: rgba(0, 77, 64,  255);  border: 1px solid rgb(124, 124, 124); border-radius:5px;}</string>
                </property>
                <property name="text">
                 <string>框选区域</string>
                </property>
               </widget>
               <widget class="QPushButton" name="delButton">
                <property name="geometry">
                 <rect>
                  <x>15</x>
                  <y>90</y>
                  <width>91</width>
                  <height>31</height>
                 </rect>
                </property>
                <property name="font">
                 <font>
                  <family>微软雅黑</family>
                 </font>
                </property>
                <property name="text">
                 <string>删除</string>
                </property>
               </widget>
               <widget class="QPushButton" name="matchButton">
                <property name="geometry">
                 <rect>
                  <x>15</x>
                  <y>130</y>
                  <width>91</width>
                  <height>31</height>
                 </rect>
                </property>
                <property name="font">
                 <font>
                  <family>微软雅黑</family>
                 </font>
                </property>
                <property name="text">
                 <string>匹配</string>
                </property>
               </widget>
               <widget class="QPushButton" name="recButton">
                <property name="geometry">
                 <rect>
                  <x>15</x>
                  <y>170</y>
                  <width>91</width>
                  <height>31</height>
                 </rect>
                </property>
                <property name="font">
                 <font>
                  <family>微软雅黑</family>
                 </font>
                </property>
                <property name="text">
                 <string>识别</string>
                </property>
               </widget>
               <widget class="QPushButton" name="saveButton">
                <property name="geometry">
                 <rect>
                  <x>15</x>
                  <y>210</y>
                  <width>91</width>
                  <height>31</height>
                 </rect>
                </property>
                <property name="font">
                 <font>
                  <family>微软雅黑</family>
                 </font>
                </property>
                <property name="text">
                 <string>保存模板</string>
                </property>
               </widget>
               <widget class="QPushButton" name="savePicButton">
                <property name="geometry">
                 <rect>
                  <x>15</x>
                  <y>250</y>
                  <width>91</width>
                  <height>31</height>
                 </rect>
                </property>
                <property name="font">
                 <font>
                  <family>微软雅黑</family>
                 </font>
                </property>
                <property name="text">
                 <string>保存图片</string>
                </property>
               </widget>
               <widget class="QPushButton" name="addTransButton">
                <property name="geometry">
                 <rect>
                  <x>15</x>
                  <y>50</y>
                  <width>91</width>
                  <height>31</height>
                 </rect>
                </property>
                <property name="font">
                 <font>
                  <family>微软雅黑</family>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton:checked{background-color: rgba(38, 166, 154,  255);  border: 1px solid rgb(124, 124, 124); border-radius:5px;}
QPushButton{background-color: rgba(84, 110, 122,  255);  border: 1px solid rgb(124, 124, 124); border-radius:5px;}
QPushButton:pressed{background-color: rgba(0, 77, 64,  255);  border: 1px solid rgb(124, 124, 124); border-radius:5px;}</string>
                </property>
                <property name="text">
                 <string>框选传输区域</string>
                </property>
               </widget>
              </widget>
              <widget class="QGroupBox" name="groupBox_4">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>540</y>
                 <width>120</width>
                 <height>61</height>
                </rect>
               </property>
               <property name="title">
                <string/>
               </property>
               <widget class="QPushButton" name="captureButton">
                <property name="geometry">
                 <rect>
                  <x>10</x>
                  <y>10</y>
                  <width>101</width>
                  <height>41</height>
                 </rect>
                </property>
                <property name="font">
                 <font>
                  <family>微软雅黑</family>
                 </font>
                </property>
                <property name="text">
                 <string>检测</string>
                </property>
               </widget>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PaintLabel</class>
   <extends>QLabel</extends>
   <header>paintlabel.h</header>
  </customwidget>
  <customwidget>
   <class>CusLabel</class>
   <extends>QLabel</extends>
   <header>cuslabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
